demo/src/main/java/com/braintreepayments/api/demo/utils/PendingRequestUtil.kt

```diff
@@ -0,0 +1,54 @@
+package com.braintreepayments.api.demo.utils
+
+import android.content.Context
+import android.content.SharedPreferences
+import com.braintreepayments.api.BrowserSwitchPendingRequest
+
+class PendingRequestUtil {
+
+    companion object {
+
+        private const val SHARED_PREFS_KEY = "PENDING_REQUESTS"
+        private const val PENDING_REQUEST_KEY = "BROWSER_SWITCH_REQUEST"
+
+        fun putPendingRequest(context: Context, pendingRequest: BrowserSwitchPendingRequest.Started) {
```

Comment:
```suggestion
        fun put(context: Context, pendingRequest: BrowserSwitchPendingRequest.Started) {
```
nit: `pendingRequestStore.put()` would read nicely in the optional unwrap chains.

Reply:
Done 👍